### PR TITLE
fixed bug

### PR DESCRIPTION
**Pull request by devonpmack**
lol it was easy

### CODE REVIEW

**Comment by DavidHZhu** [comment]
I love devon yes I do

File: src/Game.js [hunk 1]
```diff
@@ -119,6 +120,8 @@ export default class Game {
       });
     });
 
+    console.log(output);
```
Comment (devonpmack) [inline src/Game.js]: oops

**Comment by devonpmack** [comment]
@DavidHZhu

File: src/Game.js [hunk 2]
```diff
@@ -56,7 +56,8 @@ export default class Game {
       creatures: this.creatures,
       avg_size: stats.avg_size,
       avg_speed: stats.avg_speed,
-      avg_distance: stats.avg_distance
+      avg_distance: stats.avg_distance,
```
Comment (DavidHZhu) [inline src/Game.js]: GG ez git diff tbh lmao

Comment (devonpmack) [reply src/Game.js]: 👍 🐜 👎

**Pull request merged**